mwdb/core/karton.py

```diff
@@ -0,0 +1,93 @@
+import logging
+import shutil
+import tempfile
+
+from flask import g
+from karton.core import Config as KartonConfig
+from karton.core import Producer, Resource, Task
+from karton.core.backend import KartonBackend
+from karton.core.inspect import KartonState
+from karton.core.task import TaskPriority
+
+from .config import app_config
+
+logger = logging.getLogger("mwdb.karton")
+
+
+def get_karton_producer() -> Producer:
+    return Producer(
+        identity="karton.mwdb", config=KartonConfig(app_config.karton.config_path)
+    )
+
+
+def send_file_to_karton(file):
+    from mwdb.model.file import File
+
+    try:
+        path = file.get_path()
+        tmpfile = None
+    except Exception:
+        # If get_path doesn't work: download content to NamedTemporaryFile
+        tmpfile = tempfile.NamedTemporaryFile()
+        file_stream = file.open()
+        shutil.copyfileobj(file_stream, tmpfile)
+        File.close(file_stream)
+        path = tmpfile.name
+
+    producer = get_karton_producer()
+    feed_quality = g.auth_user.feed_quality
+    task_priority = TaskPriority.NORMAL if feed_quality == "high" else TaskPriority.LOW
+    task = Task(
+        headers={"type": "sample", "kind": "raw", "quality": feed_quality},
+        payload={
+            "sample": Resource(file.file_name, path=path, sha256=file.sha256),
+            "attributes": file.get_metakeys(as_dict=True, check_permissions=False),
+        },
+        priority=task_priority,
+    )
+    producer.send_task(task)
+
+    if tmpfile is not None:
+        tmpfile.close()
+
+    logger.info("File sent to Karton with %s", task.root_uid)
+    return task.root_uid
+
+
+def send_config_to_karton(config):
```

Comment:
```suggestion
def send_config_to_karton(config) -> str:
```
`config: Config` same situtation as above